draft-ietf-cbor-7049bis.md

```diff
@@ -1125,18 +1132,15 @@ applications but not others.
 ## Generic Encoders and Decoders {#generic}
 
 A generic CBOR decoder can decode all well-formed CBOR data and
-present them to an application.  CBOR data is well-formed if it uses
-the initial bytes, as well as the byte strings and/or data items that
-are implied by their values, in the manner defined by CBOR, and no
-extraneous data follows ({{pseudocode}}).
+present them to an application.  See {{pseudocode}}.
 
 Even though CBOR attempts to minimize these cases, not all well-formed
-CBOR data is valid: for example, the format excludes simple values
-below 32 that are encoded with an extension byte.  Also, specific tags
-may make semantic constraints that may be violated, such as by
-including a tag in a bignum tag or by following a byte string within a
-date tag.  Finally, the data may be invalid, such as invalid UTF-8
-strings or date strings that do not conform to {{RFC3339}}.  There is
+CBOR data is valid: for example, the text string `0x62c0ae` is not
```

Comment:
Is that a text string or an encoded text string?

Reply:
https://cbor-wg.github.io/CBORbis/draft-ietf-cbor-7049bis.html#majortypes is a little fuzzy on this point, but I'm happy to say "encoded text string".